src/graph/visitor/PrunePropertiesVisitor.cpp

```diff
@@ -57,7 +57,7 @@ void PrunePropertiesVisitor::visitCurrent(Project *node) {
     rootNode_ = false;
     return;
   }
-  for (auto i = 0u; i < columns.size(); ++i) {
+  for (auto i = 0u; i < columns.size() && i < colNames.size(); ++i) {
```

Comment:
revert this, And add a DCHECK.

Reply:
I think this shall be kept, because these two structures may indeed have different lengths that would crash here. Our current tests are not be able to detect all risks. DCHECK if not enough.

Reply:
Could add a DCHECK though.